client/app/components/application.coffee

```diff
@@ -105,26 +106,9 @@ module.exports = React.createClass
                 main
                     className: @props.layout
 
-                    if isAccount
-                        accountID = @state.accountID or 'new'
-                        tab = RouterGetter.getSelectedTab()
-                        key = "account-config-#{accountID}-#{tab}"
-                        AccountConfig {key, accountID, tab}
-
-                    else if @state.isEditable
-                        Compose
-                            ref                  : "compose-#{@state.action}-#{@state.messageID}"
-                            key                  : "compose-#{@state.action}-#{@state.messageID}"
-                            id                   : @state.messageID
-                            action               : @state.action
-                            message              : message
-                            inReplyTo            : RouterGetter.getReplyMessage @state.messageID
-                            settings             : SettingsStore.get()
-                            account              : RouterGetter.getAccount()
-
-                    else
-                        div
-                            className: 'panels'
+                    div
```

Comment:
If I understand completely `new account` behaviour, default mailbox should appear below `new account` modal; BUT there is an issue here because emails from this mailbox are not leaded because `RouterStore._action` is not `Mailbox.SHOW_ALL`